mama/c_cpp/src/c/listenermsgcallback.c

```diff
@@ -86,11 +86,13 @@ listenerMsgCallback_create( listenerMsgCallback *result,
     msgCallback* callback = (msgCallback*)calloc( 1, sizeof( msgCallback ) );
 
 #ifdef WITH_ENTITLEMENTS  /* No listener creation without a client. */
```

Comment:
Please revert all changes in this file or we'll get merge conflicts when the entitlements stuff lands, and it's not relevant to publisher events.